sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for \
+easy large numeric calculations, but which also can be easily programmed for \
+difficult or long calculations. It can accept a command line \
+argument, in which case it executes that single command and exits. \
+Otherwise, it enters interactive mode. In this mode, it accepts \
+commands one at a time, processes them, and displays the answers. \
+In the simplest case, commands are simply expressions which are evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2014  Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc$secondaryArchSuffix-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	calc$secondaryArchSuffix = $portVersion
+	cmd:calc$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:awk
+	cmd:sed
+	cmd:diff
+	cmd:egrep
+	cmd:sort
+	cmd:tee
+	cmd:ctags
+	cmd:chmod
+	cmd:fmt
+	cmd:xargs
+	cmd:cmp
+	cmd:mkdir
+	cmd:rm
+	cmd:touch
+	cmd:rmdir
+	cmd:cp
+	cmd:mv
+	cmd:cat
+#	cmd:col
+	cmd:less
+	cmd:ln
+	cmd:strip
+	"
+
```

Comment:
To fix the issue about the hard links, you can insert this block here:
```
PATCH()
{
	sed -i -e 's/^LN= ln$/& -s/' Makefile
}
```
You could also avoid this method and, instead, change the "**`LN= ln`**" into "**`LN= ln -s`**" in the Makefile and regenerate the patchset.
Feel free to chose whatever method you prefer. Both are OK ;-)